marketplaceItems/sitPoint/appResources/appData/sitClient.js

```diff
@@ -43,186 +43,152 @@
 
     // #endregion DEPENDENCIES
 
-    // #region SITTABLE OVERLAY - Overlay on chair "Click to Sit" or "Trigger to Sit"
-
-    // Constants
-    var OVERLAY_SITTABLE_FADE_MS = 50; // "Click/Trigger to Sit" overlay fade after 50 ms
-    var OVERLAY_SITTABLE_DISTANCE_SHOW_M = 3; // m 
-    var OVERLAY_SITTABLE_DISTANCE_MAX_M = 5; // m
-    var OVERLAY_SITTABLE_START_ALPHA = 0.7;
-    var OVERLAY_SITTABLE_MIN_END_ALPHA = 0.075; // fades to this alpha value
-    var OVERLAY_URL_SITTABLE_HMD = Script.resolvePath("./resources/images/triggerToSit.png");
-    var OVERLAY_URL_SITTABLE_DESKTOP = Script.resolvePath("./resources/images/clickToSit.png");
-
-    // Dynamic variables
-    var overlaySittable = null;
-    var intervalSittableTransparency = null;
-
-    // Create sittable overlay
+    // #region SITTABLE - Local entity image on chair "Click to Sit" or "Trigger to Sit"
+
+    // Create sittable local entity
+    var SITTABLE_FADE_MS = 50; // "Click/Trigger to Sit" local entity image fade after 50 ms
+    var SITTABLE_DISTANCE_SHOW_M = 3; // m 
+    var SITTABLE_DISTANCE_MAX_M = 5; // m
+    var SITTABLE_START_ALPHA = 0.7;
+    var SITTABLE_MIN_END_ALPHA = 0.075; // fades to this alpha value
+    var SITTABLE_IMAGE_URL_HMD = Script.resolvePath("./resources/images/triggerToSit.png");
+    var SITTABLE_IMAGE_URL_DESKTOP = Script.resolvePath("./resources/images/clickToSit.png");
+    var sittable_y_offset = 0.01; // default 0.01
```

Comment:
```suggestion
    var SITTABLE_Y_OFFSET = 0.01; // default 0.01
```
(and change where it's referenced)